.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ defaults: &defaults
     GRUNTWORK_INSTALLER_VERSION: v0.0.21
     MODULE_CI_VERSION: v0.13.12
     TERRAFORM_VERSION: 0.11.11
-    TERRAGRUNT_VERSION: NONE
```

Comment:
Ah build just failed because I forgot our installer is `v` sensitive (because it looks for the tag directly). Can you update to `v0.18.3`? Sorry for the confusion!